File: Lista2/C++/Prostokat.cpp
#include <iostream>
#include <cmath>
#include "Prostokat.h"

Prostokat::Prostokat(double bok1, double bok2) 
{
    this->bok1 = bok1;
    this->bok2 = bok2;
}

double Prostokat::obwod()
{
    return ((2 * bok1) + (2 * bok2));
}

double Prostokat::pole()
{
    return (bok1 * bok2);
}

Prostokat::~Prostokat()
{
    
}

File: Lista2/Java/Prostokat.java
public class Prostokat extends Czworokat
{
    private double bok1, bok2;

    Prostokat(double bok1, double bok2)
    {
        this.bok1 = bok1;
        this.bok2 = bok2;
   }

    public double obwod()
    {
        return ((2 * bok1) + (2 * bok2));
    }

    public double pole()
    {
        return (bok1 * bok2);
    }
}